next/src/components/footer/Footer.tsx

```diff
@@ -48,6 +67,24 @@ const Footer: React.FC<{}> = () => {
           B03 CSE Building K17, UNSW
           <br />
           csesoc@csesoc.org.au
+          <br/><br/>
+          <ImagesContainer>
+            <a href="https://discord.gg/AM4GB5zuB6">
+              <Image src={DiscordLogo} alt="Discord" />
+            </a>
+            <a href="https://www.facebook.com/csesoc/">
+              <Image src={FacebookLogo} alt="Facebook" />
+            </a>
+            <a href="https://www.instagram.com/csesoc_unsw/?hl=en">
+              <Image src={InstagramLogo} alt="Instagram" />
+            </a>
+            <a href="https://www.youtube.com/c/CSESocUNSW">
+              <Image src={YoutubeLogo} alt="Instagram" />
+            </a>
+            <a href="https://open.spotify.com/show/2h9OxTkeKNznIfNqMMYcxj">
+              <Image src={SpotifyLogo} alt="Instagram" />
+            </a>
+          </ImagesContainer>
           <br /><br />
           © 2021 — CSESoc UNSW
```

Comment:
should this be 2022?

Reply:
aa yes it should be 2022! winnie could you also update the text into 2022 when making the requested changes :pray: